assets/bin/dev

```diff
@@ -95,6 +95,9 @@ function package_dev_done() {
 
     rm -rf "vendor/${package}" "packages/${package_name}"
 
+    # Remove dangling symlinks
+    find ./app -lname "*/${package_name}" -type l -exec rm {} +
```

Comment:
The laravel-base package composer.json specifies this
```
"installer-paths": {
  "app/Module/{$name}/": [
        "type:laravel-module"
  ],
"app/Domain/vendor/{$name}/": [
        "type:laravel-domain"
      ]
 }
```
together with this:
```
 "repositories": {
    "package-dev": {
      "type": "path",
      "url": "./packages/*/",
      "options": {
        "symlink": true
      }
    }
  }
```
In order to provide Modules/Domains from packages. The package_dev_done function removes the package but not the symlink to the package leaving dangling symlinks. Because of this, composer can't replace it.